### DISCUSSION TITLE
CodeArea horizontal scrollbar not working

### DISCUSSION THREAD

**Discussion by Qhkr-V3** [community/community#42223] [Codespaces]
### Select Topic Area

Product Feedback

### Body

CodeArea horizontal scrollbar not working

**Comment by DavidGangel** [comment]
Horizontal scrollbar is missing and now in a PR review it's basically impossible to review a groovy file which contains long lines.
Example:
![Screenshot 2023-01-03 at 11 14 58](https://user-images.githubusercontent.com/5735394/210337926-2523478e-0476-43a6-8fa9-a9eb4e74956c.png)

Expected view and state in November was like this (just used a zoom out to be able to show what was a kind of look when it was working fine):
![Screenshot 2023-01-03 at 11 17 06](https://user-images.githubusercontent.com/5735394/210338169-a050022a-189b-4826-93bd-8e90de726704.png)

**Comment by erikaellr** [comment]
Hi @Pythonmoders and @DavidGangel, happy to have you both in the GitHub community!

From the screenshot, it looks like this feedback is about the PR experience, not Codespaces. You are more likely to get a useful response if you re-categorize this discussion post accordingly, so that someone with expertise in that area will see it. Happy coding!

**Reply by Qhkr-V3** [reply]
![Screenshot_20230106-180343__01__01](https://user-images.githubusercontent.com/100186554/211203851-9782d8e2-8767-49ac-bc74-fe95bbfcaf41.jpg)
